docs/cli.md

```diff
@@ -465,6 +559,69 @@ $ zapier logs --type=http
 ```
 
 
+## migrate
+
+> Migrates users from one version of your app to another.
+
+**Usage**: `zapier migrate FROMVERSION TOVERSION [PERCENT]`
+
+Starts a migration to move users between different versions of your app. You may also "revert" by simply swapping the from/to verion strings in the command line arguments (i.e. `zapier migrate 1.0.1 1.0.0`).
+
+Only migrate users between non-breaking versions, use `zapier deprecate` if you have breaking changes!
+
+Migrations can take between 5-10 minutes, so be patient and check `zapier history` to track the status.
+
+Note: since a migration is only for non-breaking changes, users are not emailed about the update/migration. It will be a transparent process for them.
+
+> Tip: We recommend migrating a small subset of users first, then watching error logs of the new version for any sort of odd behavior. When you feel confident there are no bugs, go ahead and migrate everyone. If you see unexpected errors, you can revert.
+
+> Tip 2: You can migrate a single user by using `--user` (i.e. `zapier migrate 1.0.0 1.0.1 --user=user@example.com`).
+
+**Arguments**
+* (required) `fromVersion` | The version FROM which to migrate users.
+* (required) `toVersion` | The version TO which to migrate users.
+* `percent` | Percentage (between 1 and 100) of users to migrate.
+
+**Flags**
+* `--user` | Migrate only this user
+* `-d, --debug` | Show extra debugging output
+
+**Examples**
+* `zapier migrate 1.0.0 1.0.1`
+* `zapier migrate 1.0.1 2.0.0 10`
+* `zapier migrate 2.0.0 2.0.1 --user=user@example.com`
+
+
+## promote
+
+> Promotes a specific version to public access.
+
+**Usage**: `zapier promote VERSION`
+
+Promotes an app version into production (non-private) rotation, which means new users can use this app version.
+
+* This [1mdoes[22m mark the version as the official public version - all other versions & users are grandfathered.
```

Comment:
In PR https://github.com/zapier/zapier-platform/pull/63, I used ANSI codes in the command help text. That's why it got misrendered here in this markdown doc. We don't have to fix it in this PR, but I just wanted to point out, before releasing 9.0.0, as part of the effort standardizing our CLI style, we should style the text properly based on the destination format. Currently, we have these formats:

* ANSI codes for the terminal. We should get rid of markdown syntax such as backticks (`` `code` ``) and double asterisks (`**bold**`) and use appropriate ANSI codes. 
* HTML
* Markdown

Reply:
Yeah, good call! I think in the docs on the commands themselves we use `chalk.bold` (etc) for nice styling. We can use [strip-ansi](https://github.com/chalk/strip-ansi) to pull them out for the markdown. 

Or (and i'm not sure how hard this is; it might just be a straight replace?) we take the ansi docs and replace the ansi with markdown and then build html out of the markdown.